script.js

```diff
@@ -45,28 +45,24 @@ const isNoteworthy = document.getElementById('isNoteworthy');
 const setEndDate = (startDate) => {
   const startTime = new Date(startDate);
   const startTimeArray = startTime.toLocaleDateString().split('/');
-
+  
   const endTime = new Date(
     startTimeArray[2],
     +startTimeArray[0] - 1,
     +startTimeArray[1] + 7,
   );
   endTimeArray = endTime.toLocaleDateString().split('/');
-  endDate.value = `${endTimeArray[2]}-${endTimeArray[0].padStart(
-    2,
-    '0',
-  )}-${endTimeArray[1].padStart(2, '0')}`;
+  endDate.value = `${endTimeArray[2]}-${endTimeArray[0].padStart(2,'0')}-${endTimeArray[1].padStart(2, '0')}`;
   if (endDate.parentElement.querySelector('em'))
     endDate.parentElement.querySelector('em').innerHTML = `${
       endTimeArray[2]
     }-${endTimeArray[0].padStart(2, '0')}-${endTimeArray[1].padStart(2, '0')}`;
 
-  const remainingDays = document.getElementById('remainingDays').children[0];
-  remainingDays.innerHTML = Math.ceil(
-    (endTime - startTime) / (1000 * 60 * 60 * 24),
-  );
-};
+/*   const remainingDays = document.getElementById('remainingDays').children[0];*/
```

Comment:
Please remove the comments

Reply:
Ok will do the changes....but now it is showing correct default remaining days (7) on your side?

Reply:
Yes Ojas